packages/website/vite.config.ts

```diff
@@ -3,4 +3,7 @@ import { defineConfig } from "vite";
 
 export default defineConfig({
   plugins: [solid()],
+  server: {
+    port: 5173
```

Comment:
Explicit Port bad

Reply:
Without this it hosts on port 3000 by default on my machine.